docusaurus/android_versioned_docs/version-5.x.x/04-compose/05-message-components/08-reactions-picker.mdx

```diff
@@ -36,18 +36,13 @@ The code above will render the following UI:
 
 ![Default ReactionsPicker Component](../../assets/compose_default_reactions_picker_component.png)
 
-:::note
-`MessagesScreen` will give you the ability to display `ReactionsPicker` only if you have defined more than 5 reactions inside of [`ChatTheme`](../03-general-customization/01-chat-theme.mdx).
```

Comment:
I think this was a mistake originally.

Tried launching it with only a single reaction and it worked fine.